packages/engine/Source/Scene/Fog.js

```diff
@@ -32,8 +40,41 @@ function Fog() {
    * Decreasing the value will push the fog further from the viewer, but decrease performance as more of the terrain is rendered.
    * @type {number}
    * @default 2.0e-4
+   * @example
+   * // Double the default fog density
+   * viewer.scene.fog.density = 0.0004;
    */
   this.density = 2.0e-4;
+  /**
+   * A scalar used in the function to adjust density based on the height of the camera above the terrain.
+   * @type {number}
+   * @default 0.001
+   */
+  this.fogScalar = 0.001;
+  /**
+   * Exponent factor used in the function to adjust density based on the height of the camera. Higher values will lead
+   * to more dense fog at lower heights and a slower falloff as you get higher.
+   * Positive values only, if the value is negative it will be clamped to 0;
```

Comment:
```suggestion
   * Exponent factor used in the function to adjust how density changes based on the height of the camera above the ellipsoid. Smaller values produce a more gradual transition as camera height increases.
   * Value must be greater than 0.
```

Totally borrowing from [Unreal's doc](https://dev.epicgames.com/documentation/en-us/unreal-engine/exponential-height-fog-in-unreal-engine) here.